src/lib/dfp/render-advert.ts

```diff
@@ -146,6 +146,38 @@ const addContentClass = (adSlotNode: HTMLElement) => {
 	}
 };
 
+/* Centre certain slots in their containers, this is class is added dynamically to avoid rendering quirks with the ad label, and variable width ads. */
```

Comment:
```suggestion
/* Centre certain slots in their containers, this class is added dynamically to avoid rendering quirks with the ad label, and variable width ads. */
```